shippable_adapter.py

```diff
@@ -45,11 +45,29 @@ def __post(self, url, body):
         try:
             data = json.dumps(body)
         except Exception as exc:
-            trace = traceback.format_exc()
-            error = '{0}: {1}'.format(str(exc), trace)
-            self.log.error('POST {0} failed with error: {1}'.format(
-                url, error))
-            return True, error
+            # if there is an error, test each console line and
+            # replace the bad line with an error message
+            new_job_console_models = []
+            for console in body['jobConsoleModels']:
```

Comment:
`post_job_consoles` might be a better place to do this. Will need another check to see if body is a string or an object.